pom.xml

```diff
@@ -2001,15 +2001,26 @@
                 <param>com.puppycrawl.tools.checkstyle.DetailNodeTreeStringPrinter</param>
                 <param>com.puppycrawl.tools.checkstyle.AstTreeStringPrinter</param>
                 <param>com.puppycrawl.tools.checkstyle.TreeWalker</param>
-                <param>com.puppycrawl.tools.checkstyle.Main</param>
               </targetClasses>
               <targetTests>
                 <param>com.puppycrawl.tools.checkstyle.DetailNodeTreeStringPrinterTest</param>
                 <param>com.puppycrawl.tools.checkstyle.AstTreeStringPrinterTest</param>
                 <param>com.puppycrawl.tools.checkstyle.TreeWalkerTest</param>
-                <param>com.puppycrawl.tools.checkstyle.MainTest</param>
+                <param>com.puppycrawl.tools.checkstyle.checks.blocks.*</param>
```

Comment:
You said why you removed `Main` but why do we need to add all these checks?
Shouldn't all `TreeWalker` tests be in `TreeWalkerTest`?